sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,7 @@
+select "g". "name" as "genre",
+      count ("cm".*)
+  from "castMembers" as "cm"
+join "filmGenre" using ("filmId")
+join "genres" as "g" using ("genreId")
+where "actorId" = 178
```

Comment:
For this exercise, please make sure you're only using the information available to you, not info you need to obtain through another query (manual or otherwise):

---

![image](https://user-images.githubusercontent.com/16143771/186732786-acd63075-9a38-46a6-895a-9dc08d30e708.png)